packages/client/src/index.test.ts

```diff
@@ -251,6 +286,19 @@ describe('LogtoClient', () => {
       expect(storage.getItem('LOGTO_SESSION_MANAGER')).toBeUndefined();
     });
 
+    test('unknown state in url should fail', async () => {
+      const storage = new MemoryStorage();
+      const logto = await LogtoClient.create({
+        domain: DOMAIN,
+        clientId: CLIENT_ID,
+        storage,
+      });
+      await logto.loginWithRedirect(REDIRECT_URI, jest.fn());
+      await expect(
+        logto.handleCallback(`${REDIRECT_CALLBACK_WITHOUT_STATE}&state=xxxx`)
```

Comment:
why `xxxx` is an unknown state?
mismatched state test is needed

Reply:
Invalid.

- There is no `mismatched state` unit test above.
- `unknown state` means `wrong state`. 
    - I used to `wrong state` to describe this test.
    - simeng recommend me to use `unknown state` in login context.